### ISSUE TITLE
Reconcile sensor and level between Kvalobs and LARD

### ISSUE THREAD

**Issue by Lun4m** [metno/lard#41]
A missing (sensor, level) in the Obsinn message format is treated differently by Kvalobs and LARD.

| Obsinn param code | Kvalobs Sensor Level | LARD Sensor Level |
| :-------------: | :-------------: | :-------------: |
| TA | `('0', 0)` | `(NULL, NULL)` |
| TA(0,0) | `('0', 0)` | `(0, 0)` |

While migrating from Kvalobs it's impossible to tell the difference between the two cases, unless the timeseries is already present in LARD. And after the migration, we might end up with duplicate timeseries.

**Comment by ketilt** [comment]
Will it help to have a tool that dedupes cases of suspected duplication? That is, if similar timeseries are checked to be fully identical, then the "least preferred" one will be removed and archived?

**Comment by Lun4m** [comment]
Yes, that's the idea, unless we decide to change how LARD works in this regard
